docs/book/src/reference/metrics.md

```diff
@@ -73,6 +76,29 @@ for the metrics exported from the namespace where the project is running
+
+| Metrics name                                  | Type      | Description                                                                                                                                                                                                                          |
+| --------------------------------------------- | :-------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| workqueue_depth                               | Gauge     | Current depth of workqueue.                                                                                                                                                                                                          |
+| workqueue_adds_total                          | Counter   | Total number of adds handled by workqueue.                                                                                                                                                                                           |
+| workqueue_queue_duration_seconds              | Histogram | How long in seconds an item stays in workqueue before being requested.                                                                                                                                                               |
+| workqueue_work_duration_seconds               | Histogram | How long in seconds processing an item from workqueue takes.                                                                                                                                                                         |
+| workqueue_unfinished_work_seconds             | Gauge     | How many seconds of work has been done that is in progress and hasn't been observed by work_duration. Large values indicate stuck threads. One can deduce the number of stuck threads by observing the rate at which this increases. |
+| workqueue_longest_running_processor_seconds   | Gauge     | How many seconds has the longest running processor for workqueue been running.                                                                                                                                                       |
+| workqueue_retries_total                       | Counter   | Total number of retries handled by workqueue.                                                                                                                                                                                        |
+| rest_client_requests_total                    | Counter   | Number of HTTP requests, partitioned by status code, method, and host.                                                                                                                                                               |
```

Comment:
@Kavinjsir this one I could not find.
From where do you check it out?

Reply:
Hey, @camilamacedo86 .
I think it's here:
https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/metrics/client_go_adapter.go#L79